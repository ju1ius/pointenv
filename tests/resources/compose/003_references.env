a=1
b=${a}
c=${d:-b}
d=${f:-${d:-${h:-42}}}

f=${f:-'g'$a"h"}
g="${g:-h$a'i'}"
# escaped dollar triggers exapnsion in unquoted values
h=\${a}
