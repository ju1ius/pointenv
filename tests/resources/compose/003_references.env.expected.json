[
  {
    "key": "a",
    "value": "1"
  },
  {
    "key": "b",
    "value": "1"
  },
  {
    "key": "c",
    "value": "b"
  },
  {
    "key": "d",
    "value": "42"
  },
  {
    "key": "f",
    "value": "'g'1\"h\""
  },
  {
    "key": "g",
    "value": "h1'i'"
  },
  {
    "key": "h",
    "value": "\\1"
  },
  {
    "key": "i0",
    "value": "$~"
  },
  {
    "key": "i1",
    "value": "$~"
  },
  {
    "key": "k0",
    "value": "$0"
  },
  {
    "key": "k1",
    "value": "$(pwd)"
  }
]
